PostGreSQLNotifyExtensibleDependency/README.md

```diff
@@ -0,0 +1,159 @@
+
+On the PostgreSQL database side, you need a trigger that invokes a trigger function. By specifying that the trigger should be called AFTER UPDATE and AFTER DELETE, the trigger function sends a JSON payload on a channel. 
+
+Any clients LISTENING on the channel using the previous LISTEN command get the payload and processes it using the information given therein. In our case, we take the payload and if the data matches the information about the cached item the dependency is attached to, we invoke the *DependencyChanged* event handler.
+
+### Sample Application Structure
+
+A sample ***NotifyExtensibleDependency*** implementation given in the sample follows the approach given in the previous section and you can view the source code [here](./src/PostGreSQLNotificationDependency/PostGreSQLDependency.cs).
+
+Also given in this project are the scripts for creating the [demo customers table](./Resources/CustomerTable.sql), the [trigger](./Resources/CustomerTableTrigger.sql) to invoke the trigger function for notification and the [trigger function](./Resources/CustomerTriggerFunction.sql) itself. 
+
+Besides these, there are two console applications that reference the [***NotifyExtensibleDependency***](./src/PostGreSQLNotificationDependency/) .NET Standard 2.0 library. One is [NETConsoleUI](./src/NETConsoleUI/) which is a .NET Framework 4.7.2 application and the other is [NETCOREConsoleUI](./src/NETCOREConsoleUI/) which is a .NET Core 3.1 application.
+
+These two console applications are identical in their source code and are meant to show that both a .NET Core client and a .NET framework client can make use of the same ***NotifyExtensibleDependency*** feature.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- The latest version of NCache Enterprise is installed. The latest version can be gotten from [here](https://www.alachisoft.com/download-ncache.html). If you are using a Linux cache server, download the Enterprise .NET Core installation. For a Windows cache server, you can install either the .NET Framework or the .NET Core installation.
+
+  **NOTE**: The rest of this article assumes you have installed the .NET Framework installation although the steps for the .NET Core application are exactly the same.
+
+- In your PostgreSQL database, run the SQL ***CREATE*** scripts given in the [Resources](./Resources/) folder to create the *customers* table, the *customers* table trigger that will be invoking the Notifications trigger function, and the trigger function itself.
+
+  **IMPORTANT**:Confirm that the trigger is enabled.
+
```

Comment:
Make sure the relevant ports are open to allow for full communication between NCache and the PostgreSQL Database. 
This sample was tested with PostgreSQL version "10.15"